crates/lib/src/execution_data.rs

```diff
@@ -343,4 +346,83 @@ mod describe_readout_map {
         ]);
         assert_eq!(ro, expected);
     }
+
+    #[test_case(0, 0 => Some(ReadoutValue::Integer(1)) ; "returns value when both indices are in bounds")]
+    #[test_case(1, 0 => None ; "returns None when row index is off by 1")]
+    #[test_case(0, 1 => None ; "returns None when column index is off by 1")]
```

Comment:
Is this a new syntax that compares the return value for you? 🤯

Reply:
Yeah, it's pretty amazing.